### DISCUSSION TITLE
import/Export MySQL Backup

### DISCUSSION THREAD

**Discussion by erfjab** [Gozargah/Marzban#1128] [General]
# Import

برای ایمپورت کردن بکاپ دیتابیس MySQL می‌توانید از دستور زیر استفاده کنید.

```
docker exec -i marzban-mysql-1 mysql -u root -p"DB_PASSWORD" marzban < "SQL_BACKUP_ADDRESS"
```

متغیر ‌DB PASSWORD رو با رمز عبور دیتابیس پنل‌تون جایگزین کنید. متغیر SQL_BACKUP_ADDRESS با آدرسی که فایل .sql بکاپ‌تون قرار داره جایگزین کنید تا بکاپ‌تون ایمپورت بشود. سپس بعد این کار یکبار مرزبان رو ریستارت کنید.

# Export

برای اکسپورت کردن بکاپ دیتابیس MySQL فقط کافیه دستور را برعکس کنید. 

```
docker exec -i marzban-mysql-1 mysql -u root -p"DB_PASSWORD" marzban > "/root/Backup.sql"
```

اینجاهم متغیر DB PASSWORD رو با رمز عبور دیتابیس پنل‌تون جایگزین کنید. متغیر SQL_BACKUP_ADDRESS با آدرسی که میخوایید بکاپ‌تون ذخیره بشه جایگزین کنید. 

این موارد رو به داکیومنت رسمی مرزبان اضافه کردم. 
سوالی و راهنمایی نیاز داشتید بپرسید.

**Comment by Eslender73** [comment]
> # Import
> برای ایمپورت کردن بکاپ دیتابیس MySQL می‌توانید از دستور زیر استفاده کنید.
> 
> ```
> docker exec -i marzban-mysql-1 mysql -u root -p"DB_PASSWORD" marzban < "SQL_BACKUP_ADDRESS"
> ```
> 
> متغیر ‌DB PASSWORD رو با رمز عبور دیتابیس پنل‌تون جایگزین کنید. متغیر SQL_BACKUP_ADDRESS با آدرسی که فایل .sql بکاپ‌تون قرار داره جایگزین کنید تا بکاپ‌تون ایمپورت بشود. سپس بعد این کار یکبار مرزبان رو ریستارت کنید.
> 
> # Export
> برای اکسپورت کردن بکاپ دیتابیس MySQL فقط کافیه دستور را برعکس کنید.
> 
> ```
> docker exec -i marzban-mysql-1 mysql -u root -p"DB_PASSWORD" marzban > "/root/Backup.sql"
> ```
> 
> اینجاهم متغیر DB PASSWORD رو با رمز عبور دیتابیس پنل‌تون جایگزین کنید. متغیر SQL_BACKUP_ADDRESS با آدرسی که میخوایید بکاپ‌تون ذخیره بشه جایگزین کنید.
> 
> این موارد رو به داکیومنت رسمی مرزبان اضافه کردم. سوالی و راهنمایی نیاز داشتید بپرسید.

این دستور رو میزنم این وارنینگ رو میده و گیر میکنه تهشم بکاپ نمیگیره

mysql: [Warning] Using a password on the command line interface can be insecure.
